components/NewResourceComment.js

```diff
@@ -0,0 +1,37 @@
+import { OpenAPIProvider } from "react-openapi-client";
+import React from "react";
+import { Button, Modal, Box } from "@mui/material";
+import NewResourceCommentForm from "./NewResourceCommentForm";
+
+function NewResourceComment({ resourceId }) {
+  const [open, setOpen] = React.useState(false);
```

Comment:
It is better to import implicitly the useState() function. https://reactjs.org/docs/hooks-state.html

Reply:
Done!